### PR TITLE
More favicons

### PR DESCRIPTION
**Pull request by jamesatheyDDS**
Addresses #205.

## Checklist

I have…

- [x] run the application locally (`bin/rails server`) and verified that my changes behave as expected.
- [x] run static code analysis (`bin/rubocop`) and vulnerability scan (`bin/brakeman`) against my changes.
- [x] run the test suite (`bin/rake spec`) and verified that all tests pass.
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x] thoroughly outlined below the steps necessary to test my changes.
- [x] attached screenshots illustrating relevant behavior before and after my changes.

## Summary of Changes

- Uses a simplified version of the USTRANSCOM logo with stronger lines and no text for the favicon files at 32x32 and below.
- Adds Android Chrome icons.
- Adds MS tile icons (generated by realfavicongenerator.net)
- Adds safari mask SVG, used for pinned tabs and the touch bar
- Adds iOS icon.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `more-favicons`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in your Web browser of choice.

### favicon in Chrome tab before

screen shot 2017-12-01 at 12 16 22 pm

### favicon in Chrome tab after

screen shot 2017-12-01 at 12 26 45 pm

### Safari

screen shot 2017-12-01 at 12 39 28 pm

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,8 +12,16 @@
   <meta name="theme-color" content="#d6d7d9">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <%= tag.link href: asset_path('ustranscom-emblem-256x256.png'), rel: 'apple-shortcut-icon' %>
-  <%= tag.link href: asset_path('ustranscom-emblem-256x256.png'), rel: 'shortcut icon' %>
+  <!-- PNG versions of favicon.ico -->
+  <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
+  <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
+  <!-- iOS homescreen icon -->
+  <link rel="apple-touch-icon" href="/apple-touch-icon.png">
```
Comment (jamesatheyDDS) [inline app/views/layouts/application.html.erb]: [Some advocate generating every possible resolution under the sun for iOS](https://css-tricks.com/favicon-quiz/). That sounded like a lot of work. 180x180 will have to suffice (rescaled automatically) for everyone.

**Comment by jamesatheyDDS** [comment]
@jgarber623-gov I've added the -precomposed and resolution-specific apple touch icons, to cover the other 404s we've been seeing.

**Comment by Elliottdotgov** [comment]
👍

**Comment by aileendds** [comment]
lgtm

**Pull request merged**